### PR TITLE
Small edit to radio ra-dec example

### PR DESCRIPTION
**Pull request by hayesla**
just text edit - clarifying how the LOFAR image was produced following discussion with @eoincarley

### CODE REVIEW

File: examples/units_and_coordinates/radec_to_hpc_map.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 How to create a `~sunpy.map.Map` in Helioprojective Coordinate Frame from radio observations
 in GCRS (RA-DEC).
 
-In this example a LOFAR FITS file (made from CASA) is read in, the WCS
-header information is then used to make a new header with the information in
-Helioprojective, and a `~sunpy.map.Map` is made.
+In this example a LOFAR FITS file (created LOFAR's Default Pre-Processing Pipeline (DPPP) and
+WSClean Imager,  see `here <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`_) is read in,
```
Comment (nabobalis) [inline examples/units_and_coordinates/radec_to_hpc_map.py:10]:
```suggestion
WSClean Imager(`see here <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`__) is read in,
```
?

**Review by nabobalis — APPROVED** [review]

File: examples/units_and_coordinates/radec_to_hpc_map.py (hunk 1 above)
Comment (hayesla) [reply examples/units_and_coordinates/radec_to_hpc_map.py:10]: yeah maybe this link doesn't need to be here? @eoincarley who created the FITS file for this example suggests we include it.

Comment (nabobalis) [reply examples/units_and_coordinates/radec_to_hpc_map.py:10]: I am happy with the link.

Comment (nabobalis) [inline examples/units_and_coordinates/radec_to_hpc_map.py:10]:
```suggestion
In this example a LOFAR FITS file (created with LOFAR's `Default Pre-Processing Pipeline (DPPP) and
WSClean Imager <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`__) is read in,
```

File: examples/units_and_coordinates/radec_to_hpc_map.py [hunk 2]
```diff
@@ -6,9 +6,10 @@
 How to create a `~sunpy.map.Map` in Helioprojective Coordinate Frame from radio observations
 in GCRS (RA-DEC).
 
-In this example a LOFAR FITS file (made from CASA) is read in, the WCS
-header information is then used to make a new header with the information in
-Helioprojective, and a `~sunpy.map.Map` is made.
+In this example a LOFAR FITS file (created with LOFAR's `Default Pre-Processing Pipeline (DPPP) and
+WSClean Imager` <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`__) is read in,
```
Comment (nabobalis) [inline examples/units_and_coordinates/radec_to_hpc_map.py:10]:
```suggestion
WSClean Imager <https://support.astron.nl/LOFARImagingCookbook/dppp.html>`__) is read in,
```

Comment (hayesla) [reply examples/units_and_coordinates/radec_to_hpc_map.py]: 🤦

**Pull request merged**